fhir-persistence-schema/src/test/java/com/ibm/fhir/schema/derby/OldFhirResourceTableGroup.java

```diff
@@ -115,11 +110,10 @@ public class OldFhirResourceTableGroup {
     /**
      * Public constructor
      */
-    public OldFhirResourceTableGroup(PhysicalDataModel model, String schemaName, SessionVariableDef sessionVariable,
+    public OldFhirResourceTableGroup(PhysicalDataModel model, String schemaName,
```

Comment:
This helps show why I was hoping that I'd get a chance to replace this one with some setup that deploys the old schema with a previously-related version of the schema tool (see class-level comments).

However, I think its been handy to be able to run this without an explicit setup and so maybe we should just remove those class-level comments and continue to maintain this special flavor of the old FhirResourceTableGroup for migration test purposes.

Thoughts?

Reply:
This has been pretty handy for me to catch migration issues at build time (which makes making changes quicker (along with the more recent "old" schema version). I've updated the class comment.